### PR TITLE
[Windows] win2022: enable use of test-signed code

### PR DESCRIPTION
**Pull request by al-cheb**
# Description
https://docs.microsoft.com/en-us/windows-hardware/drivers/install/the-testsigning-boot-configuration-option#behavior-of-windows-when-loading-test-signed-code-is-enabled

Behavior of Windows when loading test-signed code is enabled

- When loading test-signed code is enabled, Windows does the following:
- Displays a watermark with the text "Test Mode" in all four corners of the desktop, to remind users the system has test-signing enabled. Note  Starting with Windows 7, Windows displays this watermark only in the lower right-hand corner of the desktop.
- Displays a watermark with the text "Test Mode" in the lower-left corner of the desktop to remind users that the system has test-signing enabled.
- The operating system loader and the kernel load drivers that are signed by any certificate. The certificate validation is not required to chain up to a trusted root certification authority. However, each driver image file must have a digital signature.

image

#### Related issue:
https://github.com/actions/virtual-environments/issues/1637

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by miketimofeev — APPROVED** [review]

File: images/win/windows2022.json [hunk 1]
```diff
@@ -143,7 +151,7 @@
         },
         {
             "type": "windows-restart",
-            "restart_timeout": "10m"
+            "restart_timeout": "30m"
```
Comment (nibanks) [inline images/win/windows2022.json:154]: Just curious, why this change?

**Review by nibanks — APPROVED** [review]
Looks good to me. Just wondering why you had to increase a restart timeout.

File: images/win/windows2022.json (hunk 1 above)
Comment (al-cheb) [reply images/win/windows2022.json:154]: During the image generation several attempts were unsuccessful due to short reboot timeout: `vhd: Timeout waiting for machine to restart.`

Comment (nibanks) [reply images/win/windows2022.json:154]: Ok. Thanks! Interesting that test-signing caused that. Oh well!
